### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by lionel-faber**
Update dependencies to resolve security vulnerabilities.

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
Thanks for the pull request, and welcome! The MaidSafe team is excited to review your changes, and you should hear from @krishnaIndia (or someone else) soon.

If any changes to this PR are deemed necessary, please add them as extra commits. This ensures that the reviewer can see what has changed since they last reviewed the code. The way GitHub handles out-of-date commits, this should also make it reasonably obvious what issues have or haven't been addressed. Large or tricky changes may require several passes of review and changes.

Please see [CONTRIBUTOR.md](../blob/master/CONTRIBUTOR) for more information.

File: safe_web_api_playground/package.json [hunk 1]
```diff
@@ -21,20 +21,19 @@
...
     "babelify": "^7.3.0",
     "body-parser": "^1.17.2",
-    "bootstrap": "^4.1.3",
+    "bootstrap": "^4.3.1",
     "browserify": "^14.4.0",
     "codemirror": "^5.33.0",
     "envfile": "^2.1.1",
     "express": "^4.15.3",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.0",
```
Comment (hunterlester) [inline safe_web_api_playground/package.json]: The major version change breaks `gulp.start` operations in `gupfile.js`. Needs updating.

File: safe_web_api_playground/package.json [hunk 2]
```diff
@@ -21,20 +21,19 @@
...
     "gulp-concat": "^2.6.1",
     "gulp-image": "^2.9.0",
     "gulp-nodemon": "^2.2.1",
     "gulp-sass": "^3.1.0",
     "gulp-watch": "^4.3.11",
     "jquery": "^3.2.1",
     "npm-font-open-sans": "^1.1.0",
-    "safe_ux_guidelines": "https://github.com/maidsafe/safe_ux_guidelines.git",
```
Comment (hunterlester) [inline safe_web_api_playground/package.json]: In addition to removing `safe_ux_guidelines`, those areas in the app which relied on it need to be refactored such as `index.html` and the local `scss` files.

**Review by hunterlester — CHANGES_REQUESTED** [review]
Styles are broken and gulp is not functioning as expected.
If you want I can pull your branch and open up a pull request with the additional changes needed.

**Comment by lionel1704** [comment]
> Styles are broken and gulp is not functioning as expected.
That's strange. They seemed okay to me. Here are the steps I followed.
- Removed `node_modules` inside the web API playground
- Ran `yarn` to install the packages
- `yarn gulp` served the app on `localhost:3003`
I even uploaded the build dir via WHM and the published sited seemed fine too.
Am I going wrong somewhere?

File: safe_web_api_playground/package.json (hunk 2 above)
Comment (lionel1704) [reply safe_web_api_playground/package.json]: From what I observed in the gulpfile the `safe-styles` gulp task inserts the safe styles from the UX guidelines repo into the `static/scss` folder on which the `sass()` operation is performed to transform them into css files which are linked from `index.html`.

**Comment by hunterlester** [comment]
@lionel1704 Most likely, is that yarn has cached `safe_ux_guidelines`, so it may be continuing to install for you even though the repo has been deleted.

**Comment by lionel1704** [comment]
I did a `yarn cache clean` and followed the same steps again. Still seems okay to me. @manavbp gave it a try on his machine and it seemed okay there too. 

If it's still breaking for you, feel free to raise a PR to my branch.

**Comment by hunterlester** [comment]
@lionel1704 Okay I'll open a PR on your branch.
This is what my styles look like:
image

What are the contents of you `build` directory?

**Comment by lionel1704** [comment]
Here you go [build.zip](https://github.com/maidsafe/safe_examples/files/2989272/build.zip)

**Comment by hunterlester** [comment]
@lionel1704 Thank you. `main.css` still contains `scss` from the no longer existing `safe_ux_guidelines`.
Did you delete build directory and then run `yarn gulp` again?

**Comment by hunterlester** [comment]
@lionel1704 Looking good! Thanks, Lionel.
Last thing is to update the gulpfile: https://github.com/lionel1704/safe_examples/pull/1

**Review by hunterlester — APPROVED** [review]

**Comment by lionel1704** [comment]
@S-Coyle can we have this merged please?

**Pull request merged**